#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_config.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Several Debian specific changes to config file

@DPATCH@
diff -urNad privoxy~/config privoxy/config
--- privoxy~/config
+++ privoxy/config
@@ -1,4 +1,4 @@
-#        Sample Configuration File for Privoxy v3.0.6
+#        Sample Configuration File for Privoxy
 #
 #  $Id: config,v 1.56 2006/11/14 01:54:36 hal9 Exp $
 #
@@ -123,7 +123,7 @@
 #          If set, this option should be the first option in the config
 #          file, because it is used while the config file is being read.
 #
-#user-manual http://www.privoxy.org/user-manual/
+user-manual /usr/share/doc/privoxy/user-manual
 
 #
 #  1.2. trust-info-url
@@ -159,8 +159,8 @@
 #      don't end up locked out from the information on why they were
 #      locked out in the first place!
 #
-trust-info-url  http://www.example.com/why_we_block.html
-trust-info-url  http://www.example.com/what_we_allow.html
+#trust-info-url  http://www.example.com/why_we_block.html
+#trust-info-url  http://www.example.com/what_we_allow.html
 
 #
 #  1.3. admin-address
@@ -266,7 +266,7 @@
 #      flat, except for confdir/templates, where the HTML templates
 #      for CGI output reside (e.g. Privoxy's 404 error page).
 #
-confdir .
+confdir /etc/privoxy
 
 #
 #  2.2. logdir
@@ -293,7 +293,7 @@
 #
 #      No trailing "/", please
 #
-logdir .
+logdir /var/log/privoxy
 
 #
 #  2.3. actionsfile
@@ -735,7 +735,7 @@
 #      Note that you must have compiled Privoxy with support for this
 #      feature, otherwise this option has no effect.
 #
-enable-remote-toggle  1
+enable-remote-toggle  0
 
 #
 #  4.4. enable-remote-http-toggle
@@ -803,7 +803,7 @@
 #      Note that you must have compiled Privoxy with support for this
 #      feature, otherwise this option has no effect.
 #
-enable-edit-actions 1
+enable-edit-actions 0
 
 #
 #  4.6. ACLs: permit-access and deny-access
